conf/local.conf.sample

```diff
@@ -9,9 +9,6 @@ BB_NUMBER_THREADS ?= "${@oe.utils.cpu_count()}"
 PARALLEL_MAKE ?= "-j ${@oe.utils.cpu_count()}"
 
 PACKAGE_CLASSES ?= "package_rpm"
-EXTRA_IMAGE_FEATURES_append = " debug-tweaks "
```

Comment:
@rpannek valid point.
The IMAGE definitions will be found in the image recipe (https://github.com/Pelagicore/meta-pelux/pull/13/commits/5f87c0f37a89d80575f0708da4df5e0d4c9d4ee1#diff-b6bafc97f66a201ce881d55c0b4998fcR27), while the DISTRO definitions are found in the distro .conf file.